quickstart-services.yml

```diff
@@ -183,7 +183,7 @@ services:
       - 'host.docker.internal:host-gateway'
     container_name: alkemio_dev_notifications
     hostname: notifications
-    image: alkemio/notifications:v0.3.2
+    image: vyanakiev/alkemio-notifications:v0.4.6
```

Comment:
update before finalization...

Reply:
I need to run the whole flow first to see whether I need extra changes there so I can update + release notifications and use it here.